samples/process-document.v1beta3.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2020, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main(projectId, location, processorId, filePath) {
+  // [START documentai_process_document]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const location = 'YOUR_PROJECT_LOCATION'; // Format is 'us' or 'eu'
+  // const processor = 'YOUR_PROCESSOR_ID'; // Create processor in Cloud Console
+  // const filePath = '/path/to/local/pdf';
+
+  const {
+    DocumentProcessorServiceClient,
+  } = require('@google-cloud/documentai').v1beta3;
+
+  const clientOptions = {
+    apiEndpoint: 'us-documentai.googleapis.com',
+  };
+
+  const client = new DocumentProcessorServiceClient(clientOptions);
+
+  async function processDocument() {
+    // The full resource name of the processor, e.g.:
+    // projects/project-id/locations/location/processor/processor-id
+    // You must create new processors in the Cloud Console first
+    const name = `projects/${projectId}/locations/${location}/processors/${processorId}`;
+
+    // Read the file into memory.
+    const fs = require('fs');
+    const imageFile = fs.readFileSync(filePath);
```

Comment:
I really like to avoid using `fs.readFileSync` methods (or any sync methods for that matter) in our samples.  Folks tend to copy/paste this code, and you should *never* use those sync methods in production.  Instead, could I trouble you to use `fs.promises`?
```js
const fs = require('fs/promises');
```

Reply:
Sure thing. We should also probably change the guidance here, too:
https://cloud.google.com/vision/docs/base64#node.js

Reply:
Done.